src/components/helpers/pretty-text-input.js

```diff
@@ -452,14 +450,6 @@ export default createReactClass({
   },
 
   onChangeAndTagCodeMirror() {
-    if (!this.codeMirror) {
-      // We might have erased our codemirror instance before hitting the trailing
-      // end of the debounce. If so, get the value out of state.
-      if (this.props.value !== this.state.value) {
-        this.onChange(this.state.value);
-      }
-      return;
-    }
```

Comment:
This won't be invoked anymore when the CM instance has died. I'm not sure of the exact implication of not invoking `onChange` here anymore though. I've done a QA locally and to me, everything seemed ok. But again, not sure I didn't miss something.

Reply:
Pretty sure you can't just remove this. I'm not quite sure when this would happen and not blur per the other problem I mentioned, but it's the same kind of thing I was fixing in this commit:

https://github.com/zapier/formatic/commit/00d0766324afd47f0fe5e52e67cafdf4b08b6f38

If the component gets unmounted while debouncing, we lose the value that's in the state, which means that value won't get saved by the surrounding app.